src/internal/converters/ics/ics.go

```diff
@@ -443,12 +438,50 @@ func updateEventProperties(ctx context.Context, event models.Eventable, iCalEven
 		iCalEvent.AddAttendee(addr, props...)
 	}
 
+	// LOCATION - https://www.rfc-editor.org/rfc/rfc5545#section-3.8.1.7
 	location := getLocationString(event.GetLocation())
 	if len(location) > 0 {
 		iCalEvent.SetLocation(location)
 	}
 
-	// TODO Handle different attachment type (file, item and reference)
+	// X-MICROSOFT-LOCATIONDISPLAYNAME (Outlook seems to use this)
+	loc := event.GetLocation()
+	if loc != nil {
+		locationDisplayName := ptr.Val(event.GetLocation().GetDisplayName())
+		if len(locationDisplayName) > 0 {
```

Comment:
aside: i feel like we should modify `ptr.ValOK` to return OK only if the value was non-nil and non-zero.  We never really use it for non-nil only checks, but we have a lot of non-zero checks.  Thoughts?

Reply:
Hmm, interesting thought. I personally almost always have used `Val` and not much of `ValOK`. That said, I'm not sure if adding an empty check will help much as empty can also be technically "valid".